src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -163,4 +175,25 @@ protected function getIntlDateFormatter()
 
         return $intlDateFormatter;
     }
+
+    /**
+     * Checks if the pattern contains only a date.
+     *
+     * @param string $pattern The input pattern
+     *
+     * @return bool
+     */
+    protected function isPatternDateOnly()
+    {
+        // the default pattern includes time
+        if (null === $this->pattern) {
+            return false;
+        }
+
+        // strip escaped text
```

Comment:
I don't know - ICU escaping is not very common and it might not be clear what we're doing here.